### PR TITLE
feat: Support customizing connection name for all commands

### PR DESCRIPTION
**Pull request by setu4993**
This PR adds a `connection-name` arg to all commands.

This also downgrades the Django version to v1.11.29, which _might_ break something but I'm still trying to figure out what it'll break.

### CODE REVIEW

File: vmigration_helper/management/commands/migration_rollback.py [hunk 1]
```diff
@@ -48,24 +48,23 @@ def add_arguments(self, parser):
         )
 
         parser.add_argument(
-            '--dry-run',
-            action='store_true',
-            help=f'Show the commands that would have run but do not run them'
+            "--migrate-cmd",
+            default=MIGRATE_COMMAND,
+            help=f'The migration command template (accepts {{app}} and {{name}}). Default is: "{MIGRATE_COMMAND}"',
```
Comment (setu4993) [inline vmigration_helper/management/commands/migration_rollback.py]: VS Code's settings formatted these to be more compatible with `black`. I kept the changes I added to be minimal but if it's a problem, I can revert all of these style changes.

File: pyproject.toml [hunk 2]
```diff
@@ -12,8 +12,8 @@ exclude = ["main"]
 keywords = ["django","migration","rollback"]
 
 [tool.poetry.dependencies]
-python = ">=3.6,<4.0"
-Django = "^3.2.6"
+python = ">=3.7,<4.0"
```
Comment (setu4993) [inline pyproject.toml:15]: The latest versions of `wheel` do not support py36, only py37+, so I had to change this.

File: vmigration_helper/management/commands/migration_rollback.py (hunk 1 above)
Comment (setu4993) [reply vmigration_helper/management/commands/migration_rollback.py]: Never mind, I reverted those.

**Comment by bluedenim** [comment]
Thanks for the PR. I wasn't able to pull it down to make some refactoring (the common code for the connection-name param).

My PR (based on your work here) is https://github.com/bluedenim/vmigration-helper/pull/15

**Comment by setu4993** [comment]
Closing in favor of #15.

**Pull request closed without merging**